server/__init__.py

```diff
@@ -20,6 +22,17 @@
     debug_toolbar
 )
 
+class HashidConverter(BaseConverter):
+    # ID hashing configuration.
+    # DO NOT CHANGE ONCE THE APP IS PUBLICLY AVAILABLE. You will break every
```

Comment:
Should we add a test to make sure that 
 - the hashid converter exists
 - that the config hasn't changed (by encoding/decoding a value) and comparing it's value.